tests/meta/upload/test_meta_upload_scoutapi.py

```diff
@@ -3,35 +3,73 @@
 
 import pytest
 import yaml
+
 from cg.apps.hk import HousekeeperAPI
 from cg.meta.upload.scoutapi import UploadScoutAPI
 from cg.store import Store
 
 
-def test_generate_config_adds_rank_model_version(analysis: Store.Analysis, upload_scout_api:
-                                                 UploadScoutAPI):
+# pylint: disable=protected-access
+def test_unlinked_family_is_linked(upload_scout_api):
+    """Test that is_family check fails when samples are not linked"""
+    # GIVEN a upload scout api and case data for a family without linked individuals
+    family_data = {
+        "samples": [
+            {"sample_id": "ADM2", "father": "0", "mother": "0"},
+            {"sample_id": "ADM3", "father": "0", "mother": "0"},
+        ]
+    }
+    # WHEN running the check if family is linked
+    res = upload_scout_api._is_family_case(family_data)
+    # THEN assert that the test returns False
+    assert res is False
+
+
+# pylint: disable=protected-access
```

Comment:
same here

Reply:
I guess we want to test privat methods. The alternative would be to let the linter complain like we do in many other places, especially in the tests. I think that this was a nice way to solve the linting problem here

Reply:
There is a nice discussion [here](https://stackoverflow.com/questions/105007/should-i-test-private-methods-or-only-public-ones) if you want to dig into this topic